data/xml/2023.nlposs.xml

```diff
@@ -288,28 +288,27 @@
-      <author><first>William</first><last>Tjhi</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
-      <author><first>David</first><last>Ong</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
-      <author><first>Peerat</first><last>Limkonchotiwat</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
+      <author><first>David</first><last>Ong</last><affiliation>AI Singapore</affiliation></author>
+      <author><first>Peerat</first><last>Limkonchotiwat</last><affiliation>Vidyasirimedhi Institute of Science and Technology, Thailand</affiliation></author>
       <pages>245-245</pages>
       <abstract>Abstract. (not necessary)</abstract>
       <url hash="632dc5a1">2023.nlposs-1.26</url>
       <bibkey>tjhi-etal-2023-sea</bibkey>
     </paper>
     <paper id="27">
       <title>trl<fixed-case>X</fixed-case>: A Framework for Large Scale Open Source <fixed-case>RLHF</fixed-case></title>
-      <author><first>Louis</first><last>Castricato</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
+      <author><first>Louis</first><last>Castricato</last><affiliation>EleutherAI</affiliation></author>
       <pages>246-246</pages>
-      <abstract>Abstract. (not necessary)</abstract>
+      <abstract>Reinforcement learning from human feedback (RLHF) utilizes human feedback to better align large language models with human preferences via online optimization against a learned reward model. Current RLHF paradigms rely on Proximal Policy Optimization (PPO), which quickly becomes a challenge to implement and scale up to large architectures. To address this difficulty we created the trlX library as a feature-complete open-source framework for RLHF fine-tuning of models up to and exceeding 70 billion parameters. We implemented support for multiple types of distributed training including distributed data parallel, model sharded, as well as tensor, sequential, and pipeline parallelism.</abstract>
       <url hash="b8184f0c">2023.nlposs-1.27</url>
       <bibkey>castricato-2023-trlx</bibkey>
     </paper>
     <paper id="28">
       <title>Towards Explainable and Accessible <fixed-case>AI</fixed-case></title>
-      <author><first>Brandon</first><last>Duderstadt</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
-      <author><first>Yuvanesh</first><last>Anand</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
+      <author><first>Brandon</first><last>Duderstadt</last><affiliation>Nomic AI</affiliation></author>
+      <author><first>Yuvanesh</first><last>Anand</last><affiliation>Virginia Institute of Technology & Nomic AI</affiliation></author>
```

Comment:
XML requires this to be written as `&amp;`

Reply:
Thanks! Fixed that in the new commit.